### ISSUE TITLE
Example React App using an Axis camera 

### ISSUE THREAD

**Issue by mark-klein1** [AxisCommunications/media-stream-library-js#393]
I have few Axis cameras and want to use the AxisCommunications/media-stream-library-js and React to develop a surveillance system with multiple cameras.

I cloned your github repo on MacOS High Sierra 10.13.6 and was able to run all the yarn commands without problems (yarn, build, dev, examples)

The camera use for testing is AXIS M3037 Network Camera.

The still image works fine, but **the video does not show** .. so tried modifying the source code on the project without success.

If I use these links on the browser http://192.168.2.106//axis-cgi/mjpg/video.cgi and http://192.168.2.106//axis-cgi/jpg/image.cgi they both work and show the still image **and the video.** 

I changed all the authorized variables in APP.jsx to true, just in case because if I use the link http://192.168.2.106/axis-cgi/usergroup.cgi in the browser the reply is:
**The requested URL /axis-cgi/usergroup.cgi was not found on this server.**
and also changed the setting in the camera to allow access without userid/password.

Also changed the format="H264" in App.jsx to format="MJPEG" because I have never been able to make this camera work with h264 even with the h264 plugin active in Firefox. 

I only use this camera with the Axis Live View utility  provided by Axis Communications, it only works with the setting Motion-JPEG stream profile, and as an ONVIF video source on a propietary surveillance system.

This is what the **console in the browser shows** with the link http://localhost:3554/ and setting to mjpeg **(the still image works fine):**

client:52 [WDS] Live Reloading enabled.
openwebsocket.js:34 WebSocket connection to 'ws://192.168.2.106/rtsp-over-websocket' failed: Error during WebSocket handshake: Unexpected response code: 404
Promise @ openwebsocket.js:34
exports.openWebSocket @ openwebsocket.js:32
open @ index.js:109
Html5CanvasPipeline @ html5-canvas-pipeline.js:45
(anonymous) @ WsRtspCanvas.js:122
commitHookEffectListMount @ react-dom.development.js:19731
commitPassiveHookEffects @ react-dom.development.js:19769
callCallback @ react-dom.development.js:188
invokeGuardedCallbackDev @ react-dom.development.js:237
invokeGuardedCallback @ react-dom.development.js:292
flushPassiveEffectsImpl @ react-dom.development.js:22853
unstable_runWithPriority @ scheduler.development.js:653
runWithPriority$1 @ react-dom.development.js:11039
flushPassiveEffects @ react-dom.development.js:22820
(anonymous) @ react-dom.development.js:22699
workLoop @ scheduler.development.js:597
flushWork @ scheduler.development.js:552
performWorkUntilDeadline @ scheduler.development.js:164
openwebsocket.js:71 GET http://localhost:3554/axis-cgi/rtspwssession.cgi?1588134585986 404 (Not Found)
ws.onerror @ openwebsocket.js:71
openwebsocket.js:48 failed to load token 404 <!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /axis-cgi/rtspwssession.cgi</pre>
</body>
</html>

onLoadToken @ openwebsocket.js:48
localhost/:1 Uncaught (in promise) 

In summary .. **What should I do to make the example App work correctly :) ?**

**Comment by mark-klein1** [comment]
Update:

I **upgraded the firmware to the lastest version** and everything seems to be working fine now.
Even h264 is working in the browsers.
The examples all work correctly.

All tests also passed, here is the result:
Test Suites: 26 passed, 26 total
Tests:       154 passed, 154 total
Snapshots:   0 total
Time:        81.71s
Ran all test suites.
A worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --runInBand --detectOpenHandles to find leaks.
✨  Done in 93.21s.
